installers/windows/minikube.nsi

```diff
@@ -0,0 +1,182 @@
+# This installs two files, minikube.exe and logo.ico, creates a start menu shortcut, builds an uninstaller, and
+# adds uninstall information to the registry for Add/Remove Programs
+
+# To get started, put this script into a folder with the two files (minikube.exe, logo.ico, and LICENSE.txt -
+# You'll have to create these yourself) and run makensis on it
+
+# If you change the names "minikube.exe", "logo.ico", or "LICENSE.txt" you should do a search and replace - they
+# show up in a few places.
+# All the other settings can be tweaked by editing the !defines at the top of this script
+# Unicode true # This command is not available in 2.46 which is on apt-get. Debating how to proceed
```

Comment:
This will just cause some languages to not appear correctly.  This is probably okay to start with but I would be sure not to lose track of this (maybe file a bug) as it will be a bad UX for some users.